platform/darwin/src/MGLStyleLayer_Private.h

```diff
@@ -5,9 +5,20 @@
 
 #include <mbgl/style/layer.hpp>
 
+@class MGLMapView;
+
 @interface MGLStyleLayer (Private)
 
 @property (nonatomic, readwrite, copy) NSString *identifier;
 @property (nonatomic) mbgl::style::Layer *layer;
```

Comment:
For future reference, I think we should add documentation for this property like we did in `MGLSource_Private.h` for the `rawSource` property. When coming back to this code (or looking at it for the first time) it is not immediately obvious that this is a raw pointer that is useable even after layer ownership is transferred in `addToMapView:`. Also, for consistency, it might be worth considering renaming this to `rawLayer` (or naming `[MGLSource rawSource]` to `[MGLSource source]` (again)).